pkg/image/prune/imagepruner.go

```diff
@@ -94,6 +96,9 @@ type PrunerOptions struct {
 	// KeepTagRevisions is the minimum number of tag revisions to preserve;
 	// revisions older than this value are candidates for pruning.
 	KeepTagRevisions int
+	// KeepOverSizeLimitImages indicates that images exceeding defined limits (openshift.io/Image)
+	// will not be considerate as candidates for pruning.
```

Comment:
`s/considerate/considered/`

The sentence suggests that oversized images won't be pruned no matter what if set to `true`. The name of this option is also unfortunate. Oversized images will be candidates for pruning if too old or deprecated by newer tag revisions.

Can I suggest one another name: `--prune-over-sized`?